src/components/document-metadata-modal.yml

```diff
@@ -22,9 +22,21 @@ invalidTags:
 tagsPlaceholder:
   en: Enter keywords
   de: Stichwörter angeben
-useTemplateDocument:
-  en: Use default template
-  de: Standardvorlage benutzen
+tagsInfo:
+  en: Tags make the document discoverable when searching by words that match them
+  de: Tags make the document discoverable when searching by words that match them [but in German]
```

Comment:
This description sounds very technical, maybe we could be more casual like:

Here you can specify search terms with which your document can be found by other users
Hier kannst du Suchbegriffe angeben, mit denen dein Dokument von anderen Nutzern gefunden werden kann

Reply:
Sounds better, changed!